test/tools/router_check/test/config/ClusterHeader_expected.json

```diff
@@ -0,0 +1,23 @@
+[
```

Comment:
suggestion. rename these files to _golden.json or ClusterHeader.json.golden or something of that sort? Its upto you.